test/templates/icons/index.njk

```diff
@@ -0,0 +1,293 @@
+{% extends "layout.njk" %}
+
+{% block content %}
+<!-- find original template at https://github.com/shakyShane/gulp-svg-sprites/blob/master/tmpl/preview.html -->
+<style>
+	.demo-icon-wrapper{
+		padding: 20px;
+	}
+	.demo-icon-wrapper span{
+		float: left;
+		margin-right: 20px;
+		display: inline-block;
+	}
+</style>
+
+<div class="demo-icon-wrapper">
```

Comment:
@saadmhybris This is the generated page from `icons.js` — this can be tested at http://localhost:3030/icons in the test app. I added into the sec template the `extends` and `block` directives.